packages/react/src/file-upload/docs/overview.mdx

```diff
@@ -8,12 +8,17 @@ relatedComponents: ['file-input']
 ---
 
 ```jsx live
-<FileUpload
-	label="Avatar image"
```

Comment:
Rather than having "Avatar image" as the label, I've changed the examples on this page to "Upload documents". This is more consistent with what we have in Figma andStorybook.

I've also updated the example to accept multiple files, which is one of the primary reasons you would use this component over File input.